File: cs_pdf2/example/FmMenu.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace example
{
    public partial class FmMenu : Form
    {
        public FmMenu()
        {
            InitializeComponent();
        }

        private void BtnExample1_Click(object sender, EventArgs e)
        {
            Example1.Run();
        }

        private void BtnOpenOutput_Click(object sender, EventArgs e)
        {
            System.Diagnostics.Process.Start("output\\");
        }

    }
}

File: vb_pdf2/example/FmMenu.vb
Public Class FmMenu

    Private Sub BtnExample1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles BtnExample1.Click
        Example1.Run()
    End Sub

    Private Sub BtnOpenOutput_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles BtnOpenOutput.Click
        System.Diagnostics.Process.Start("output\")
    End Sub

End Class
